crates/shell/src/shells/snapshots/starbase_shell__shells__fish__tests__formats_cd_hook.snap.new

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
Since these are `.new` snaps, seems we forgot to include them. Can you run `cargo insta approve` and commit them.